docs/launch_micro_tests.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+#This is a job launch script for boot tests
+
+import os
+import sys
+from uuid import UUID
+
+from gem5art.artifact.artifact import Artifact
+from gem5art.run import gem5Run
+from gem5art.tasks.tasks import run_gem5_instance
```

Comment:
Too many tasks?

Reply:
I don't think it will work without two tasks.tasks